pkg/network/netlink/circuit_breaker.go

```diff
@@ -114,13 +114,7 @@ func (c *CircuitBreaker) update(now time.Time) {
 	}
 
 	lastUpdate := c.lastUpdate.Load()
-	deltaInSec := float64(now.UnixNano()-lastUpdate) / float64(time.Second.Nanoseconds())
-
-	// This is to avoid a divide by 0 panic or a spurious spike due
-	// to a reset followed immediately by an update call
```

Comment:
Could you retain this comment please?

Reply:
Oh yeah for sure it's not great that the tool removed comments 😬 
I think it's because the comment is between the variable declaration and the if block
I'll add it back ! Sorry about that